test/common.test.js

```diff
@@ -137,71 +139,89 @@ each([1, 2], function(version) {
         it('should include userId', function() {
           analytics.user().identify('id');
           analytics.track('Completed Order', {});
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             user_id: 'id'
           });
         });
 
         it('should include orderId', function() {
-          analytics.track('Completed Order', { id: 1 });
-          analytics.calledOnce(window.__adroll.record_user, {
+          analytics.track('Completed Order', { orderId: 1 });
+          analytics.called(window.__adroll.record_user, {
             adroll_segments: 'order_created',
             order_id: 1
           });
         });
 
         it('should map revenue for normal track events', function() {
           analytics.track('Teems', { revenue: 17.38 });
-          analytics.calledOnce(window.__adroll.record_user, {
-            adroll_segments: 'Ate habanero cheese',
+          analytics.called(window.__adroll.record_user, {
+            adroll_segments: 'ate_habanero_cheese',
```

Comment:
I believe that's just how AdRoll names the segments so when I first added this sample data I forgot about that.